pnpm-workspace.yaml

```diff
@@ -0,0 +1,3 @@
+packages:
+  - addon
+  - test-app
```

Comment:
**nitpick**

At this stage, we don't have the `test-app` package yet. I'd have added line 3 in #1062. We can merge this PR as is, since (I think) `pnpm` ignores entries corresponding to packages that don't exist yet.

Reply:
**nitpick**

In #1060, I noticed you had to update the `volta` entry in the root `package.json`.

https://github.com/emberjs/ember-qunit/blob/5862d31192101b62b9092eac68d20f41714d3902/package.json#L15-L18

I'm guessing that, in this pull request, the entry should have similarly been updated again, by replacing `yarn` with `pnpm`? (Maybe this happens in a later PR. At any rate, we can merge the PR as is.)

Reply:
Nice catch! Thanks!
For conflict management, i'd like to do that in the test-app extraction pr.

Reply:
> At this stage, we don't have the test-app package yet.

I missed this comment.

I extracted code from PR #1062 -- this this #1063, is meant for easier review of the whole process only, and isn't itself an atomic change.